src/app/in-viewport/window/window-ref.service.ts

```diff
@@ -23,6 +23,8 @@ export class WindowRef {
   opener = null;
   outerHeight = null;
   outerWidth = null;
+  pageXOffset = null;
+  pageYOffset = null;
   pageXOffsetleft = null;
```

Comment:
I think `pageYOffsetleft` and `pageXOffsetleft` are incorrect and shouldn't be here, it should be `pageXOffset` and `pageYOffset` so maybe you can just replace them instead.

Reply:
👍 Updated